index.js

```diff
@@ -13,11 +13,17 @@ program
   .description(pkg.description);
 
 program
-  .command('top')
+  .command('top [articleCount]')
   .description('List Linda Ikeji Top Stories')
-  .option('-n, --number <int>", "specify number of stories')
-  .action(function(options){
-    var count = isNaN(parseInt(options) || parseInt(options) == 0) ? 20 : options;
+  .action(function(articleCount) {
+    var count = parseInt(articleCount) || 20;
+
+    // Prevent negative values
+    if (count < 0) {
+      count = 20;
+    }
+
+    // Todo: consider using a logger, to enable setting the verbosity of the program
     console.log("List top " + count + " Linda Ikeji Stories");
```

Comment:
Using a logger is awesome, but in this case it might just seem like an overhead, so using `console.log` is just simply fine